projects/ngx-form-object/src/lib/interfaces/form-error.interface.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Remove this rule as the interfaces are not prefixed at the moment. Down the line, we might prefix them, but that should be done for all the interfaces at once.